File: 250704/수 등장 횟수/number-frequency.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const [n, m] = input[0].split(' ').map(Number);
const arr = input[1].split(' ').map(Number);
const queries = input[2].split(' ').map(Number);

const map = new Map();

arr.forEach(val => {
    map.set(val, (map.get(val) ?? 0) + 1)
})
const answer = [];
queries.forEach(val => {
    answer.push(map.get(val)?? 0)
})

console.log(answer.join(' '))

File: 250826/수 등장 횟수/number-frequency.py
n, m = map(int, input().split())
arr = list(map(int, input().split()))
nums = list(map(int, input().split()))

temp = {}

for e in arr:
    if e not in temp:
        temp[e] = 1
    else:
        temp[e] +=1


for e in nums:
    if e not in temp:
        print(0, end=' ')
    else:
        print(temp[e], end=' ')